features/step_definitions/registry.rb

```diff
@@ -371,3 +371,22 @@
   cb_name ||= :registry_route
   cb[cb_name] = api_hostname.gsub("api","registry")
 end
+
+# Generate default route
+Given /^I enable image-registry default route$/ do
+  ensure_admin_tagged
+  step 'I run the :patch client command with:', table(%{
+      | resource      | configs.imageregistry.operator.openshift.io |
+      | resource_name | cluster                                     |
+      | p             | {"spec":{"defaultRoute":true}}              |
+      | type          | merge                                       |
+  })
+  step %Q/the step should succeed/
+end
+
+Given /^default (default-route) route is stored in the#{OPT_SYM} clipboard$/ do |route_name, cb_name|
```

Comment:
IMO, (default-route)  can  replace with #{QUOTED}

Reply:
it is presenty hardcoding route name to `default-route`. It is not clear from step name that it is getting route from registry namespace. I would suggest a step definition:
```
Given /^default image registry route is stored in the#{OPT_SYM} clipboard$/ do |cb_name|
```
then you can hardcode route name and project name inside the step body.

Reply:
Good idea, I have changed, thanks!